textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/dictionarymatcher/DictionaryMatcherTest.java

```diff
@@ -53,7 +53,7 @@
     private IDataWriter dataWriter;
     private IDataReader dataReader;
     private Analyzer analyzer;
```

Comment:
I saw many variables defined here.  Do we need all of them?  Can we move some of them to the functions that really need them?  Another operator "KeywordMatcher" initially had a similar pattern.  Later they were able to move some variables into functions.